### DISCUSSION TITLE
Projects Need Non-Issue Entries

### DISCUSSION THREAD

**Discussion by hughesjs** [community/community#17680] [Projects and Issues]
Actions on a project might not belong to any particular repository or to multiple repositories.

We need a way to capture and assign these too!

**Comment by gertv** [comment]
You have "draft" issues in the new projects beta and while you can use them for this kind of thing, it would be nice to have an option to have a plain "task" card type as well.

**Comment by robertandrews** [comment]
I think, or thought, this, too. But maybe it's just where I am in my learning curve on Issues more generally.

Is it that GitHub is being opinionated on the feature for good reason - that it really thinks Issues should be the container for project tasks? It's the language of "Draft" that got me... if I have no intention on making an item into an Issue, that language is off-putting...

BUT - it may just be that this implication forces me into a workflow where I use Issues a lot more. Which may be a good thing.

**Comment by caseybaggz** [comment]
I would love for Projects to allow new "views" to not carry over the issues from another and allow the use of "notes" on the boards like they did in classic. If they offered this, I would convert - but right now, beta isn't fulfilling a base need like classic does.

For example, we use classic as 2 projects (which could now be a single one in beta if they add these missing features):

* board 1 - our weekly goal/sprint/etc. Specifically tied to issues (beta covers) ✅ 
* board 2 - our yearly Roadmap that only contains notes (beta missing feature) ❌ 

Our boards:
https://github.com/pluralsight/tva/projects
